metrics/task.go

```diff
@@ -90,6 +110,9 @@ func (m *metricsTask) Execute(ctx context.Context) error {
 			if err := m.Subsystem.StatsDClient().Timing(queueLatencyMetricNameHist, timeElapsed, m.Subsystem.Options.Tags, 1); err != nil {
 				util.Warnf("unable to submit metric: %v", err)
 			}
+			if err := m.Subsystem.StatsDClient().Gauge("faktory.jobs.latency", float64(timeElapsed.Milliseconds()), tags, 1); err != nil {
```

Comment:
Interesting that this is originally a `Timing`. What is that? Is that a histogram? I wonder if this one should be a histogram too (instead of or in addition to a gauge).

Reply:
This metric has always been a `Guage`, see line 107.

We also submitted a `Timing` which is a superset of a histogram: https://github.com/statsd/statsd/blob/master/docs/metric_types.md#timing
I didn't think we've ever used this metric, but it turns out you've used it on the Hubble dashboard so I'll re-add it.